apps/common/src/main/scala/org/lfdecentralizedtrust/splice/util/SpliceUtil.scala

```diff
@@ -249,6 +249,16 @@ object SpliceUtil {
     ).asJava,
   )
 
+  // TODO(#2251): set the extra steps to the empty list once the config in the live system is using an empty list as well
```

Comment:
why does that matter? i.e. what do we lose by setting it to an empty list immediately?

Reply:
ack. will do. Issue created here to look into the SV UI: https://github.com/hyperledger-labs/splice/issues/2264